src/WebUI/Components/PodsComponent.tsx

```diff
@@ -12,6 +12,7 @@ import { Utils } from "../Utils";
 import { StatusSize, Status } from "azure-devops-ui/Status";
 import { Tooltip } from "azure-devops-ui/TooltipEx";
 import { PodStatusComponent } from "./PodStatusComponent";
+import "azure-devops-ui/Components/TooltipEx/Tooltip.scss";
```

Comment:
this is not included in the original devops, i will open an issue with our devops repo.